### ISSUE TITLE
Feature Addition: Implement Resume Download Option for Portfolio Website

### ISSUE THREAD

**Issue by kartikey-codes** [adrianhajdin/project_3D_developer_portfolio#140]
Hello there! I'm excited to propose an enhancement for the portfolio website – adding a resume download option. This addition will provide visitors with the opportunity to easily access and download resume. Kindly assign this task to me for implementation. Looking forward to contributing to the project! Thank you

**Comment by xlgabriel** [comment]
Hey, take a look at what I've done in my version of this awesome portfolio. I think that's what you're asking for.
https://github.com/xlgabriel/portfolio

**Comment by kartikey-codes** [comment]
> Hey, take a look at what I've done in my version of this awesome portfolio. I think that's what you're asking for. https://github.com/xlgabriel/portfolio

Yeah, yours looks good. This is similar to what I wanted to implement. Do check out my version of the portfolio website at kartikeybhatnagar.tech
Here is the repo: https://github.com/kartikey-codes/LatestPersonalPortfolio
